karma.conf.js

```diff
@@ -0,0 +1,122 @@
+module.exports = function (config) {
+  config.set({
+
+    preprocessors: {
+      'src/main/webapp/app/**/*.js': 'coverage',
+      'src/main/webapp/app/views/**/*.html': ['ng-html2js']
```

Comment:
Does this end up including the index.html for ng-html2js?

Reply:
Works without it.